ckanext/unaids/theme/templates/package/read.html

```diff
@@ -3,6 +3,25 @@
 {% set pkg = c.pkg_dict %}
 
 {% block package_description %}
+
+  {% if pkg.org_to_allow_transfer_to %}
+    <div class="alert alert-info" style="margin-top: -10px;">
+      {% if h.check_organization_update_access(pkg.org_to_allow_transfer_to) %}
+        <p>{{ pkg.organization.title + _(' would like to transfer this dataset to your organisation') }}</p>
+        <br>
+        <a class="btn btn-default" href="{{ url_for('unaids_dataset_transfer.process_dataset_transfer', dataset_id=pkg.id) }}">
+          <i class="fa fa-refresh"></i>
+          <span> {{ _('Transfer dataset to ') +  h.get_organization(pkg.org_to_allow_transfer_to).display_name }}</span>
+        </a>
+      {% elif h.check_organization_update_access(pkg.owner_org) %}
```

Comment:
Your new auth function can be called properly with `h.check_access('organisation_editor`, {'org_id': pkg.owner_org})` I think.